### PR TITLE
JSS setup refuctoring

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: web/src/components/Button/index.tsx [hunk 1]
```diff
@@ -14,20 +12,23 @@ export interface ButtonProps {
...
-  <button
-    className={classes.button}
-    disabled={disabled}
-    type="button"
-    onClick={onClick}
-  >
-    {children}
-  </button>
-)
+const useStyles = createUseStyles(styles)
```
Comment (tobiaslins) [inline web/src/components/Button/index.tsx:15]: I would move `createUseStyles` into the styles file directly!

File: web/src/components/Button/styles.ts [hunk 2]
```diff
@@ -1,25 +1,25 @@
-import tokens from '../../tokens'
+import { JssStyle } from 'jss'
 import { ButtonProps } from './index'
 
-export default {
+export default (theme: Record<string, string>): Record<string, JssStyle> => ({
```
Comment (tobiaslins) [inline web/src/components/Button/styles.ts:4]: something like this:
```js
let useStyles = createUseStyles(theme => ({
  button: {
    background: theme.colorPrimary
  }
}))

// Using function values might be better if you have only few theme dependant styles
// and also props or state is used for other values.
useStyles = createUseStyles({
  button: {
    background: ({theme}) => theme.colorPrimary
  }
})
```

**Comment by tobiaslins** [comment]
Looks good now :)

File: web/src/components/Button/index.tsx (hunk 1 above)
Comment (AleshaOleg) [reply web/src/components/Button/index.tsx]: Just updated PR. Not sure how to name styles import - `import styles` or `import useStyles`. Used `useStyles` as it's function now

**Comment by AleshaOleg** [comment]
Also changed formating for `tokens.json`. Have no idea why it's importing them like this now

**Pull request merged**